docs/source/examples/private_utxo/overview.rst

```diff
@@ -0,0 +1,224 @@
+all who can see the Ledger.
+
+Assets in the off-Ledger state are unspent transaction outputs (UTXO) stored
+on the Ledger with a matching document held by the owner off-Ledger
+describing the Asset represented by the UTXO. Trusted Execution
+Environments(TEE) are used to validate and process off-Ledger transactions.
+Once validated by the TEE the transaction, proof of the transaction is
+submitted to the Ledger recording the UTXO consumed and the new UTXO produced.
+
+Assets represent something of value. In order to preserve this value, it is
+important that the rules governing that asset is preserved. In this system,
+users can issue assets, but only the original issuer can create more. All other
+transactions for these Assets can not change the amount of these Assets in
+circulation.
+
+In this example Asset are represented as positive whole numbers, fractional
+asset representations are not supported.
+
+The Private UTXO example manages 3 stores of information, the Asset Types,
+Holdings, and UTXO. Participants may act on these stores according to their
+permissions. Participants are identified by their public key, as such signing
+transactions indicates the participants intention to change the system.
+
+Asset Types
++++++++++++
+
+The Asset Type is the definition what exists, who created it, and how much
+of that asset is in circulation. The creator of the Asset Type may create more
+of that asset at anytime to increase the amount in circulation.
+
+Holdings
+++++++++
+
+Holdings keep a record of which participants hold how much of which Assets.
+Holdings can be thought as on-Ledger wallet or bank account where the
+balance of each Asset is held.
+
+Unspent Transaction Output (UTXO)
++++++++++++++++++++++++++++++++++
+
+When off-Ledger, the assets are represented as a document. This document
+holds the details of the assets. The sha512 hash of this document is the UTXO
+handle. In this case, the document is in a very real sense the asset that is
+being represented. Having access to the document will provide the details of
+the asset and who owns it. As such it is important that the document is kept
+secret and protected.  UtxoDocument is considered valid when there is a
+matching UTXO record in the DLT. If there is not a matching entry in the DLT
+then the Asset has either been consumed or has not yet be recorded with an
+**Convert to UTXO** or **Transfer UTXO** transactions.
+
+
+.. literalinclude:: ../../../../families/private_utxo/protos/utxo_document.proto
+    :language: protobuf
+    :lines: 18-25
+
+Trading and Transaction's
+=========================
+
+Private UTXO Transaction Family
++++++++++++++++++++++++++++++++
+
+The Private UTXO Transaction Family is implemented by a Transaction Processor
+that manages representation in the Ledger.
+
+.. toctree::
+   :maxdepth: 2
+
+   private_utxo_transaction_family.rst
+
+
+Off-Ledger Trading
+++++++++++++++++++
+
+This section describes how off-Ledger Assets stored in UtxoDocuments
```

Comment:
This edit produces an incorrect sentence: "This section describes how off-Ledger Assets are stored in UTXODocuments are traded."

perhaps we need this sentence "This section describes how off-Ledger Assets, stored in UTXODocuments, are traded."